src/port.ts

```diff
@@ -1,2 +1,12 @@
 export type Port = number;
 export type PortString = string;
+
+export type PortWithOptionalBinding =
+  | Port
```

Comment:
What's the trailing `|` do? Also below `number` -> `Port`

Reply:
Partly fixed in [8777232](https://github.com/testcontainers/testcontainers-node/pull/328/commits/8777232642285c9060c2a51f8a5604b0fba98b39). Prettier is adding the leading `|`.